pkg/gds/emails/client.go

```diff
@@ -62,6 +63,10 @@ type EmailManager struct {
 	adminsEmail  *mail.Address
 }
 
+func (m *EmailManager) ServiceEmail() *mail.Address {
+	return m.serviceEmail
+}
```

Comment:
Can you check if this accessor is still needed?

Reply:
you're right, it's no longer needed, I deleted it.